packages/typespec-ts/src/modular/buildHelperSerializers.ts

```diff
@@ -64,8 +110,33 @@ export function emitSerializerHelpersFile(
     symbolMap.set(isRecordElementSupportedFunction.symbol, sourceFile);
   }
 
+  if (!symbolMap.has(buildMultiCollection.symbol)) {
+    symbolMap.set(buildMultiCollection.symbol, sourceFile);
```

Comment:
I wonder if we can make this internal helper? modular customer doesn't need this, it's just a RLC helper.

Reply:
Agreed. These helpers are just used by the modular generated code and aren't exported for customers.